build-settings/gen-flow-types.ts

```diff
@@ -27,22 +37,15 @@ for (const inFile of files) {
         continue;
     }
 
-    const args = [
-        "flowgen",
-        inFile,
-        "-o",
-        outFile,
-        "--add-flow-header",
-        "--no-inexact",
-    ];
-
     try {
-        execFileSync("yarn", args, {cwd: rootDir});
```

Comment:
Yeah, not having to fork out to a new process through `yarn`. That makes sense that it's much faster!! Nice find.